Jenkinsfile

```diff
@@ -3,7 +3,7 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 node {
   stage ("Start") {
     // Check if the user who opened the PR is a known collaborator (i.e., has been added to a hyrise/hyrise team)
-    if (binding.hasVariable('pullRequest')) {
+    if (env.CHANGE_ID) {
```

Comment:
`env.CHANGE_ID` is only set for pull requests?

Reply:
Yes: https://github.com/jenkinsci/pipeline-github-plugin/issues/2